examples/03_drawing.rs

```diff
@@ -162,23 +162,27 @@ impl event::EventHandler<ggez::GameError> for MainState {
         // Draw a filled rectangle mesh.
         let rect = graphics::Rect::new(450.0, 450.0, 50.0, 50.0);
         canvas.draw(
-            None,
+            (), // <- equivalent to quad mesh
```

Comment:
I'm not sure I like this syntax, to be honest. Couldn't we do something like `graphics::QuadMesh` (Unit type) instead?

Reply:
It would clutter the namespace a little, but it would also be more explicit, yes.

I'd say adding `graphics::Quad` (without the "Mesh" to avoid confusion) as a unit type sounds like a good idea.